src/xdp/program.c

```diff
@@ -18,6 +18,7 @@ typedef struct _EBPF_PROG_TEST_RUN_CONTEXT {
 } EBPF_PROG_TEST_RUN_CONTEXT;
 
 typedef struct _EBPF_XDP_MD {
+    EBPF_CONTEXT_HEADER;
```

Comment:
i think this exposes a buggy typecast in the create/delete context routine, where the type provided to ebpf should be the EBPF_XDP_MD::Base field, but instead we simply casted to/from EBPF_XDP_MD. since the offsets used to be the same, the wrong typecast was never detected.

Reply:
Fixed